.travis.yml

```diff
@@ -36,7 +36,7 @@ matrix:
 env:
   global:
     # can be "stable", or anything that can be used with git checkout
-    - VALIDATOR_VERSION="8887074e5f5f48a768842ffcf96fb2952dbe0e38"
+    - VALIDATOR_VERSION="e457907ca9af4c32aede828117ea5a43737b4597"
```

Comment:
why this random commit??! the stable branch doesn't yet work?

Reply:
The bids-validator releases very often (sometimes even twice a week), so I find it more "stable" for us to regularly pin to an up to date commit.

that particular commit corresponds to my merge of the `split` rules into the validator.